girder/api/v1/group.py

```diff
@@ -247,7 +247,7 @@ def inviteToGroup(self, group, userToInvite, level, quiet, force):
                 })
                 mail_utils.sendEmail(
                     to=userToInvite['email'], text=html,
-                    subject="Girder: You've been invited to a group")
+                    subject="%s: You've been invited to a group" % SettingKey.BRAND_NAME)
```

Comment:
I was'n sure, but each time I call `ImportModel` it do a get request to the server ?
That's not slowing the app ?